src/server/footballSearch.js

```diff
@@ -0,0 +1,37 @@
+import fetch from 'isomorphic-fetch';
+import { newPromiseChain } from './../shared/utilities';
+
+const footballAccessOptions = {
```

Comment:
Please add a comment similar to in twitterSearch.js, which mentions that we're aware storing the API key here is a bad idea, but is here for simplicity.